### DISCUSSION TITLE
internal link suggestion?

### DISCUSSION THREAD

**Discussion by sky31even** [zsviczian/obsidian-excalidraw-plugin#560] [Q&A] [answered]
hello I'm new to excalidraw & its plugin for obsidian.

a little issue puzzling me is that I cant find link suggestion in this plugin while typing in `[[` . Normally if it is in a note i should expect to have list of notes to choose from and the result keep changing while i type in some charactors .

<img width="222" alt="image" src="https://user-images.githubusercontent.com/49628447/160755861-5145a58a-01ea-4bf5-9e0e-c4e89f70498b.png">

this is really troublesome if i have to copy the whole name in other place then paste here . Am i make something wrong on this function or it doesn't support this yet ? if so , any plan on it annouced ?

thank you guys~

**Comment by sky31even** [comment, marked as answer]
ok guys i've found this issue ~~

**Reply by maxdomeuhd** [reply]
Great! how did you do that? I'm struggling with the same issue

**Comment by Stefan-59** [comment]
I have the exact same question / problem.

So, is there any other solution than copy/paste the filename I had to search in advance?

Having the same comfort in getting suggestions for filenames in internal link would be a great enhancement!!

**Reply by Contrigra** [reply]
Hey, whoever looks for an answer, read this issue: https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/518
